src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -486,6 +508,17 @@ DictionaryArray SunLightFactory::get_input_metadata() const
             .insert("default", "1.0")
             .insert("help", "Light intensity multiplier"));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "size_multiplier")
+            .insert("label", "Sun Size Multiplier")
+            .insert("type", "numeric")
+            .insert("min_value", "0.0")
+            .insert("max_value", "100.0")
+            .insert("use", "optional")
+            .insert("default", "1.0")
+            .insert("help", "Shadow softness multiplier"));
```

Comment:
Maybe more explicative: "The size multiplier allows to make the sun bigger or smaller, hence making it cast softer or harder shadows."